examples/resources/mezmo_aggregate_v2_processor/resource.tf

```diff
@@ -22,15 +22,15 @@ resource "mezmo_http_source" "curl" {
   decoding    = "json"
 }
 
-resource "mezmo_aggregate_v2_processor" "processor1" {
+resource "mezmo_aggregate_processor" "processor1" {
```

Comment:
Looks like this directory name needs updated too.  I think that's causing the `Usage` to be removed from the `doc` file